doc/en/conf.py

```diff
@@ -266,18 +256,24 @@
 
 html_sidebars = {
     "index": [
-        "slim_searchbox.html",
+        "sidebar/brand.html",
+        "sidebar/search.html",
+        "sidebar/scroll-start.html",
         "sidebarintro.html",
         "globaltoc.html",
         "links.html",
-        "sourcelink.html",
+        "sidebar/scroll-end.html",
+        "style.html",
     ],
     "**": [
-        "slim_searchbox.html",
+        "sidebar/brand.html",
+        "sidebar/search.html",
+        "sidebar/scroll-start.html",
         "globaltoc.html",
         "relations.html",
         "links.html",
-        "sourcelink.html",
+        "sidebar/scroll-end.html",
+        "style.html",
```

Comment:
Oh wow, this is a bad way to do things. See https://pradyunsg.me/furo/customisation/injecting/#css-or-js for the guidance from the theme for how to inject CSS.